### DISCUSSION TITLE
A BUG of Notifications

### DISCUSSION THREAD

**Discussion by MrLuckyKing** [community/community#32380] [Other Feature Feedback, Questions, & Ideas] [answered]
like the following pic:

![](https://i.postimg.cc/13F30tmR/2022-09-09-145405-1888x778-scrot.png)

![](https://i.postimg.cc/3xbJHvtZ/2022-09-09-150204-1903x690-scrot.png)

fix it,plz.

**Comment by ludo237** [comment]
This is happening to me too. I've a repository stuck with 4 notifications but nothing appears

**Comment by radar-blip** [comment, marked as answer]
You have to mark sure _all_ notifications are marked as 'Read'. 
Go to the "Done' folder, select all, move them into inbox, select all again and mark as read.
That should do it.

**Comment by TarunavBA** [comment]
@MrLuckyKing Are you sure that you haven't marked the notifications as read?

**Comment by MrLuckyKing** [comment]
@linuxlawson 
@TarunavBA 

Yes. That's from **Unread** List. 

Not once for now.

Each times when i use **Notifications**.

And not only one repo like that.

A few repos appeared that situation.

All that repos both has **1** or **2** _Unread_ in  that. 

A kind of **RANDOM** of that.

**Not ALL** _1_ or _2_**Unread** repos appear that situation.

**Comment by jfullstackdev** [comment]
that's a common one, it's called ghost notification, this helped me twice 

https://github.com/community/community/discussions/6874#discussioncomment-2859125

**Comment by MrLuckyKing** [comment]
@jdevstatic  Thanks. I'll try it. XD
